File: code2map/docs/examples/python/output/parts/UserManagementService__validate_age.py
# code2map fragment (non-buildable)
# id: CD17
# original: docs/examples/python/user_management_service.py
# lines: 223-225
# symbol: UserManagementService#_validate_age
# notes: references dataclasses.dataclass, re, typing.Optional; calls ValueError
    def _validate_age(self, age: int) -> None:
        if age < 0 or age > 150:
            raise ValueError("年齢は0以上150以下で入力してください")

File: code2map/docs/examples/java/output/parts/UserManagementService_validateAge.java
// code2map fragment (non-buildable)
// id: CD13
// original: docs/examples/java/UserManagementService.java
// lines: 197-201
// symbol: UserManagementService#validateAge
// notes: references java.util.ArrayList, java.util.HashMap, java.util.List, java.util.Map, java.util.Optional, java.util.regex.Pattern
    private void validateAge(int age) {
        if (age < 0 || age > 150) {
            throw new IllegalArgumentException("年齢は0以上150以下で入力してください");
        }
    }
